examples/text-generation/experimental/utils.py

```diff
@@ -459,8 +459,8 @@ def setup_model(args, model_dtype, model_kwargs, logger):
 
     if args.torch_compile:
         model = get_torch_compiled_model(model, logger, args)
-        assert "PT_HPU_LAZY_MODE" in os.environ and os.environ["PT_HPU_LAZY_MODE"] == "0", (
-            "Please set PT_HPU_LAZY_MODE=0 on command line when using `--torch_compile`"
+        assert "PT_HPU_LAZY_MODE" in os.environ and os.environ["PT_HPU_LAZY_MODE"] != "1", (
```

Comment:
@jasi306 Is this assertion valid for synapse > 1.21? Does the user need to explicitly set `PT_HPU_LAZY_MODE` ?